<?xml version="1.0" encoding="UTF-8"?><record_update table="scan_table_check">
    <scan_table_check action="INSERT_OR_UPDATE">
        <active>true</active>
        <advanced>false</advanced>
        <attributes/>
        <category>manageability</category>
        <conditions table="sys_update_xml">applicationNSAMEASupdate_set.application^EQ<item endquery="false" field="application" goto="false" newquery="false" operator="NSAMEAS" or="false" value="update_set.application"/>
            <item endquery="true" field="" goto="false" newquery="false" operator="=" or="false" value=""/>
        </conditions>
        <description>Updates in the wrong update set scope may generate an error like the following when previewed: 

Cannot commit Update Set 'UPDATE_SET_NAME' because: Update scope id 'x' is different than update set scope id 'y'. Resolve the problem before committing.</description>
        <documentation_url/>
        <finding_type>scan_finding</finding_type>
        <name>Updates in wrong update set scope</name>
        <priority>2</priority>
        <resolution_details>The scope of the Customer Update [sys_update_xml] record should match the scope of the Update Set in which it resides.</resolution_details>
        <run_condition/>
        <score_max>100</score_max>
        <score_min>0</score_min>
        <score_scale>1</score_scale>
        <script><![CDATA[(function (engine) {

    // Add your code here

})(engine);]]></script>
        <short_description>Updates in wrong update set scope</short_description>
        <sys_class_name>scan_table_check</sys_class_name>
        <sys_created_by>GIYYH</sys_created_by>
        <sys_created_on>2022-10-18 14:38:55</sys_created_on>
        <sys_id>bd3aa4b287ae1150fc79edbe8bbb35fa</sys_id>
        <sys_mod_count>6</sys_mod_count>
        <sys_name>Updates in wrong update set scope</sys_name>
        <sys_package display_value="Global" source="global">global</sys_package>
        <sys_policy/>
        <sys_scope display_value="BayerInstanceScanChecks">69a9da4e874042507bd12fce8bbb35df</sys_scope>
        <sys_update_name>scan_table_check_bd3aa4b287ae1150fc79edbe8bbb35fa</sys_update_name>
        <sys_updated_by>EGMCF</sys_updated_by>
        <sys_updated_on>2024-01-31 20:50:29</sys_updated_on>
        <table>sys_update_xml</table>
        <use_manifest>false</use_manifest>
    </scan_table_check>
    <sys_claim action="INSERT_OR_UPDATE">
        <claim_owner_scope display_value="BayerInstanceScanChecks">69a9da4e874042507bd12fce8bbb35df</claim_owner_scope>
        <claim_timestamp>18d614997750000001</claim_timestamp>
        <metadata_update_name>scan_table_check_bd3aa4b287ae1150fc79edbe8bbb35fa</metadata_update_name>
        <previous_claim_app_version>1.0.0</previous_claim_app_version>
        <previous_claim_name>BayerInstanceScanChecks</previous_claim_name>
        <previous_claim_scope>69a9da4e874042507bd12fce8bbb35df</previous_claim_scope>
        <sys_created_by>EGMCF</sys_created_by>
        <sys_created_on>2024-01-31 20:50:29</sys_created_on>
        <sys_id>037ad2ce874042507bd12fce8bbb35d1</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_updated_by>EGMCF</sys_updated_by>
        <sys_updated_on>2024-01-31 20:50:29</sys_updated_on>
    </sys_claim>
</record_update>
